estate/models/__init__.py

```diff
@@ -0,0 +1,5 @@
+from . import estate_property
+from . import estate_property_type
+from . import estate_property_tag
+from . import estate_property_offer
+from . import res_users
```

Comment:
Should be sorted alphabetically. That's in the [pep8](https://peps.python.org/pep-0008/) . It's the official documentation to write clean python 😃 
```suggestion
from . import estate_property
from . import estate_property_offer
from . import estate_property_tag
from . import estate_property_type
from . import res_users
```

Reply:
Is there any automatic way that checks these format errors and fixes them maybe?
I am using pycharm now btw.

Reply:
Maybe you can do as explain [here](https://www.jetbrains.com/help/pycharm/reformat-and-rearrange-code.html#reformat-on-save)? I've never tried it.